tensorrt_llm/_torch/route_capture.py

```
@@ -0,0 +1,673 @@
+            rid = request.py_request_id
+            routes = m.assemble(rid)
+            if routes is not None:
+                if pyr._additional_generation_outputs is None:
+                    pyr._additional_generation_outputs = {}
+                pyr._additional_generation_outputs.setdefault("routed_experts", [])
+                pyr.append_additional_generation_outputs("routed_experts", routes)
+                m._attached.add(rid)
+                m._populate_prefix(request, rid)   # store this req's blocks for reuse
+                m.free(rid)   # bound memory once safely attached
+        except Exception:
+            pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)